dimagi/pages/templates/pages/sectors/agriculture_extension_programs.html

```diff
@@ -0,0 +1,97 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load dimagidotcom %}
+
+{% block title %}{% blocktrans %}
+  Agricultural Extension Programs
+{% endblocktrans %}{% endblock %}
+
+{% block body_class %}agriculture-page tucked-hero{% endblock %}
+
+{% block stylesheets %}
+  {% compress css %}
+    <link href="{% sass_src 'style/pages/sectors/agricultural.scss' %}" rel="stylesheet" media="screen" type="text/css" />
+  {% endcompress %}
+{% endblock %}
+
+{% block hero %}
+  <div class="green-theme">
+    <section class="hero hero-blog" id="hero"></section>
+  </div>
+{% endblock %}
+
+{% block content %}
+  <section>
+    {% include "sections/sectors/agriculture_extension_programs/sub_nav.html" %}
+    {% include "sections/sectors/agriculture_extension_programs/overview.html" %}
+    {% include "sections/sectors/agriculture_extension_programs/part1.html" %}
+    {% include "sections/sectors/agriculture_extension_programs/benefit.html" %}
+    {% include "sections/sectors/agriculture_extension_programs/programs.html" %}
+    {% include "sections/sectors/agriculture_extension_programs/part3.html" %}
+    {% include "sections/sectors/agriculture_extension_programs/news.html" %}
+    {% include "sections/sectors/agriculture_extension_programs/part4.html" %}
+    {% include "sections/sectors/agriculture_extension_programs/testimonial.html" %}
+  </section>  
+{% endblock %}
+
+{% block modals %}
+  <div class="modal"
+       data-dismissible="true"
+       data-size="medium"
+       id="schedule-demo-form">
+    <div class="modal-scroll">
+      <div class="modal-wrapper">
+        <a href="#" class="modal-close-button"></a>
+        <div class="content pad-horz-medium-at-medium pad-horz-line pad-vert-none green-theme">
+          <h3>
+            {% blocktrans %}
+              Schedule A Demo
+            {% endblocktrans %}
+          </h3>
+          <section id="schedule-demo-form-content"></section>
+        </div>{# .content #}
+      </div>{# .modal-wrapper #}
+    </div>{# .modal-scroll #}
+  </div>{# .modal #}
+
+  <div class="modal"
+       data-dismissible="true"
+       data-size="medium"
+       id="hubspot-download-form-pdf-summary">
+    <div class="modal-scroll">
+      <div class="modal-wrapper">
+        <a href="#" class="modal-close-button"></a>
+        <div class="content pad-horz-medium-at-medium pad-horz-line pad-vert-none green-theme">
+          <section id="download-pdf-summary-form-content"></section>
+        </div>{# .content #}
+      </div>{# .modal-wrapper #}
+    </div>{# .modal-scroll #}
+  </div>{# .modal #}
+{% endblock %}
+
+{% block js %}
+  <!--[if lte IE 8]>
+    <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2-legacy.js"></script>
+  <![endif]-->
+  <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script>
+  <script>
+    //  Schedule demo form
+    hbspt.forms.create({
```

Comment:
is there a reason why the demo form is being activated here and not through `demoForm.js`? seems like a lot of duplicated code in all of these PRs

Reply:
as I got diff formId. The given form id for this page is not matched as given in demoForm js file . So still we want to use demoForm.js or need to create new js for the same?